web/src/settings_emoji.ts

```diff
@@ -115,7 +115,11 @@ export function populate_emoji(): void {
     for (const emoji of active_emoji_data) {
         // Add people.js data for the user here.
         if (emoji.author_id !== null) {
-            emoji.author = people.get_user_by_id_assert_valid(emoji.author_id);
+            const author = people.get_user_by_id_assert_valid(emoji.author_id);
+            emoji.author = {
+                ...author,
+                is_active: people.is_person_active(author.user_id),
+            };
```

Comment:
Why not do this:

```
author.is_active = people.is_person_active(author.user_id);
emoji.author = author;
```

Reply:
the code is much more consistent and the code is more understandable in my opinion. Could you share your reasons for proposing these changes ?

Reply:
Not sure if we prefer `...` operator. Otherwise looks good!

Reply:
What about we do
```js
emoji.author.is_active = people.is_person_active(emoji.author_id)
```

Reply:
What about we do
```js
emoji.author.is_active = people.is_person_active(emoji.author_id)
```

Reply:
we are using `...` when displaying org pill
```console
(zulip-py3-venv) sujal@DESKTOP-7I7H17Q:~/zulip$ git grep "return {...creator"
web/src/stream_data.ts:    return {...creator, is_active: people.is_person_active(creator_id)};
```
but this looks more clean I guess, updated the code.